ch_lib/civitai.py

```diff
@@ -755,62 +757,44 @@ def check_models_new_version_by_model_types(model_types:list, delay:float=0.2) -
 
 
 def move_model_to_subfolder(filepath, model_info):
-    modelid = model_info["modelId"]
-
-    util.printD(f"Model ID is {modelid}")
+    model_id = model_info["modelId"]
 
-    if modelid == "":
+    if model_id == "":
         return None
 
-    content = civitai_get(f'{URLS["modelId"]}{modelid}')
-
-    util.printD(content)
+    content = civitai_get(f'{URLS["modelId"]}{model_id}')
 
     tags = content["tags"]
 
-    util.printD(tags)
-
     # iterate through tags until we find one that matches MODEL_CATEGORIES
 
     for tag in tags:
         if tag in MODEL_CATEGORIES:
             model_category = tag
 
             # create subfolder if it doesn't exist
-            util.printD("Current filepath is " + filepath)
-
             # check to make sure the model is not already in the correct subfolder
             if model_category in filepath:
-                util.printD("Model is already in the correct subfolder")
                 return filepath
 
             # get the file path without the filename
             folderpath = os.path.dirname(filepath)
 
-            util.printD("Current folderpath is " + folderpath)
-
             # create the new folder path
             new_folderpath = os.path.join(folderpath, model_category)
 
-            util.printD("New folderpath is " + new_folderpath)
-
             # create the new folder if it doesn't exist
-            if not os.path.exists(new_folderpath):
+            if not os.path.isdir(new_folderpath):
```

Comment:
I think it would be a good idea to also check if it exists. I know it's unlikely, but if somehow a file with the same name as the directory exists, `os.makedirs` would error out. In that case, it might just be better to break here and log.